docker-compose.yml

```diff
@@ -2,8 +2,6 @@ version: '2'
 services:
   web:
     build: .
-    command: java -Dserver.port=4567 -jar ./build/libs/bt-example-0.1.0.jar
+    command: java -Dserver.port=4567 -jar ./build/libs/braintree_spring_example.jar
     ports:
-      - "4567:4567"
-    volumes:
```

Comment:
This was clobbering the build folder built in the dockerfile (ref [Docker docs](https://docs.docker.com/compose/faq/#should-i-include-my-code-with-copyadd-or-a-volume)) so unless you were running the gradle task to build prior to running compose, it could never work.